addons/l10n_ar/models/account_move.py

```diff
@@ -87,7 +87,7 @@ def _get_l10n_latam_documents_domain(self):
                 domain = ['|', ('code', 'in', ['99'])] + domain
         return domain
 
-    def _check_argentinian_invoice_taxes(self):
+    def _check_argentinean_invoice_taxes(self):
```

Comment:
no change of method names in stable please

Reply:
ok @jco-odoo I've removed that change, now I think it's ok